financial-services-accelerator/internal-apis/org.wso2.financial.services.accelerator.authentication.endpoint/src/main/webapp/cookie_policy.jsp

```diff
@@ -0,0 +1,236 @@
+<%@ page import="static org.wso2.bfsi.consent.management.extensions.authservlet.utils.Utils.i18n" %><%--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ --%>
+
+<%@include file="includes/localize.jsp" %>
+
+<html>
+<head>
+    <jsp:include page="includes/head.jsp" />
+</head>
+
+<body class="sticky-footer">
+
+  <!-- header -->
+  <header class="header header-default">
+    <div class="container-fluid">
+       <div class="pull-left brand float-remove-xs text-center-xs brand-container">
+          <a href="#" title="WSO2 Open Banking">
+              <img src="images/wso2-open-banking-logo.png" class="logo" alt="WSO2 Open Banking"/>
+          </a>
+       </div>
+    </div>
+  </header>
+  
+  <div class="container-fluid static-page">
+    
+      <!-- page content -->
+      <div class="row">
+          <div class="col-xs-12 col-sm-12 col-md-3 col-lg-3">
+              <div id="toc" class="data-container"></div>
+          </div>
+          <div class="col-xs-12 col-sm-12 col-md-9 col-lg-9">
+              <div>
+                  <h1 class="wr-title gray-bg white">
+                      <%=i18n(resourceBundle, "wso2.open.banking")%> - <%=i18n(resourceBundle, "privacy.policy.cookies")%>
+                  </h1>
+              </div>
+              <!-- Customizable content. Due to this nature, i18n is not implemented for this section -->
+              <div id="cookiePolicy" class="policies-wrapper">
+                  <section>
+                      <h4><a href="https://wso2.com/solutions/financial/open-banking/"><strong>About WSO2 Open Banking Solution</strong></a></h4>
```

Comment:
are these ok to have?

Reply:
check other places in this file

Reply:
yes still we have not decided to change these